modules/install-creating-install-config-aws-local-zones.adoc

```diff
@@ -3,33 +3,61 @@
 
 :_content-type: PROCEDURE
 [id="install-creating-install-config-aws-local-zones_{context}"]
-= Modifying an installation configuration file to use AWS Local Zones subnets
+= Modifying an installation configuration file to use AWS Local Zones
 
-Modify an `install-config.yaml` file to include AWS Local Zones subnets.
+Modify an `install-config.yaml` file to include AWS Local Zones.
 
 .Prerequisites
 
-* You created subnets by using the procedure "Creating a subnet in AWS Local Zones".
+* You have configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+* You have read the configuration limitations that apply when you specify the installation program to automatically create subnets for your {product-title} cluster. See the section named "Cluster limitations in AWS Local Zones".
+* You opted in to the Local Zone group for each zone.
 * You created an `install-config.yaml` file by using the procedure "Creating the installation configuration file".
 
 .Procedure
 
-* Add the VPC and Local Zone subnets as the values of the `platform.aws.subnets` property. As an example:
+. Specify Local Zone names in the `platform.aws.zones` property of the `compute.name.edge` stanza. For example:
```

Comment:
This not sound correct:
> `compute.name.edge` stanza

Programmatically the correct term should be `compute[.name=="edge"].platform.aws.zones` or `compute[?name=="edge"].platform.aws.zones` depending on the language. What I would like to transmit is the `edge` is a value of the field `name` which is an item of the `compute` list. :)

Reply:
Thanks. Modified to:

> Modify the `install-config.yaml` file by  specifying Local Zone names in `compute[.name=="edge"].platform.aws.zones`

The `[.name=="edge"]` syntax might raise some flags by reviewing that are used to the x.y.z format. I'll put it through the review system and see what happens. Your suggestion does break away from separating the syntax and padding them with supporting words.

Reply:
OK. you can expand it if you want, something like: "...in the edge compute pool you must define the field `platform.aws.zones`..."

Reply:
Thanks. Changed to:

> Modify the `install-config.yaml` file by specifying Local Zone names in the `platform.aws.zones` property of the edge compute pool.